### PR TITLE
updating style for Submit form

### LINKED ISSUE #302: [FEATURE] update styling for Submit project form
### Description

As of now inout fields in the form taking comlete width of the page, it would be nice if we can reduce it a bit, or if we can wrap form into a card

current:- 
image

### PR DESCRIPTION
**Pull request by sdifeo**
### Fixes Issue
closes #302 
https://github.com/rohitdasu/projectmate/issues/302

### Changes proposed
As suggested, I made the form into a card.

### Screenshots
Screenshot 2022-12-30 224325

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "react-redux": "^8.0.5",
     "swr": "^2.0.0",
     "usehooks-ts": "^2.9.1",
+    "yarn": "^1.22.19",
```
Comment (JakubChorzepa) [inline package.json:43]: Hey I think you've installed your yarn package manager without using --global flag so it installed in our dependencies, please follow https://classic.yarnpkg.com/en/docs/install to install yarn. If you didn't install any packages there should be only SubmitPageForm.tsx that changed.

**Review by JakubChorzepa — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rohitdasu) [reply package.json:43]: thanks for pointing this @JakubChorzepa

File: components/SubmitPageForm/SubmitPageForm.tsx [hunk 2]
```diff
@@ -132,106 +132,110 @@ export const SubmitPageForm = () => {
...
+                    onChange={(value) => onContentChange(value, field)}
+                  />
+                );
+              }}
+            />
+          </div>
+        </div>
       </form>
       <div className="my-4 w-full">
         <Button
```
Comment (rupali-codes) [inline components/SubmitPageForm/SubmitPageForm.tsx:241]: Place this button in the center of page just like other form elements

Comment (rohitdasu) [reply components/SubmitPageForm/SubmitPageForm.tsx:241]: thanks @rupali-codes for pointing this issue.

@sdifeo the button should be inside the form container width only and it should be right aligned. let us know if you need anything. thanks

File: package.json (hunk 1 above)
Comment (sdifeo) [reply package.json:43]: Hi. Thank you for the heads up. I did not notice this. I will correct this mistake.

And that is exactly what happened. I did not know I had to install it globally. I appreciate the information.

Comment (JakubChorzepa) [reply package.json:43]: No problem, if you need any help you can join our discord server and me or someone else will surely help you.

**Comment by rohitdasu** [comment]
Screenshot 2023-01-04 at 11 07 40 PM


hi @sdifeo everything looks good except the responsiveness... could you please have a look at it? this is the screenshot for a mobile view with 320 width

**Review by JakubChorzepa — CHANGES_REQUESTED** [review]
Hi, I see you deleted yarn from dependency but still there is package-lock.json and yarn.lock is changed. Please try to remove these two files and run `yarn install` again. I think I'd solve the problem.

**Comment by rohitdasu** [comment]
@sdifeo sorry, could you please fix the conflicts. thanks

**Comment by rohitdasu** [comment]
due to inactivity, closing this PR (created a new PR for this issue). feel free to work on other issues. thanks

**Pull request closed without merging**